pom.xml

```diff
@@ -32,7 +32,7 @@
     <gson.version>2.8.5</gson.version>
     <hadoop.version>2.3.0</hadoop.version>
     <httpcomponents.version>4.5.9</httpcomponents.version>
-    <hydrator.version>2.3.0-SNAPSHOT</hydrator.version>
+    <hydrator.version>2.3.5</hydrator.version>
```

Comment:
Let's set cdap.version to 6.1.2 in line 29. hydrator-plugin v2.3.5 depends on CDAP 6.1.2 https://github.com/cdapio/hydrator-plugins/blob/v2.3.5/pom.xml#L681-L682

Reply:
Done.